bolts/fair/data/datamodules/base.py

```diff
@@ -1,17 +1,20 @@
 """Common to all datamodules."""
-from abc import abstractmethod
 import logging
-from typing import List, Optional, Sequence, Union
+from typing import Optional, Sequence, Union
 
 from kit import implements
 from kit.torch import StratifiedSampler
 import pytorch_lightning as pl
 from torch.utils.data import DataLoader, Dataset, Sampler
 
+from bolts.data.datamodules import BaseDataModule as _BaseDataModule
```

Comment:
Is this right? It looks like a circular import.

Reply:
it's not a circular import - what makes you think so? The fair modules depend on the non-fair ones, never the converse.

Reply:
oh, I missed the `.fair` part, sorry

Reply:
still, it might not be the best idea to have two identically-named classes in the code base